### PR TITLE
Blog post announcing SolvationAnalysis in JOSS

### PR DESCRIPTION
**Pull request by jennaswa**

### CODE REVIEW

**Review by richardjgowers — APPROVED** [review]
Lgtm. If @orionarcher could expand a little on what you can do with the package (aka do a little SEO here) then it'd probably get a bit more traffic through

**Comment by orionarcher** [comment]
Will do in the next few hours!

File: _posts/2023-04-20-solvationJOSS.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: SolvationAnalysis Published in JOSS
+---
+
+We're happy to announce that the Python package, [`SolvationAnalysis`][solvation], is now [published][paper] in the [Journal of Open Source Software][joss] (JOSS). Originally developed by @orioncohen as a [GSoC 2021 project][gsocblog], `SolvationAnalysis` builds on `MDAnalysis` and `pandas` to automate extraction of solvation information from molecular dynamic simulations in order to compute key solvation properties. To get started with using `SolvationAnalysis` in your workflow, check out the documentation and tutorials on [readthedocs][docs].
+
```
Comment (orionarcher) [inline _posts/2023-04-20-solvationJOSS.md:7]: > We're happy to announce that the Python package, [`SolvationAnalysis`][solvation], is now [published][paper] in the [Journal of Open Source Software][joss] (JOSS). Originally developed by @orioncohen as a [GSoC 2021 project][gsocblog], `SolvationAnalysis` builds on `MDAnalysis` and `pandas` to automate extraction of solvation information from molecular dynamic simulations in order to compute key solvation properties. To get started with using `SolvationAnalysis` in your workflow, check out the documentation and tutorials on [readthedocs][docs].

We're happy to announce that the Python package, [`SolvationAnalysis`][solvation], is now [published][paper] in the [Journal of Open Source Software][joss] (JOSS). Originally developed by @orioncohen as a [GSoC 2021 project][gsocblog], `SolvationAnalysis` builds on `MDAnalysis` and `pandas` to make analyzing solvation structure much, much easier. With a few lines of code, you can extract key solvation properties from any molecular dynamics simulation. With a few more, you can visualize solvation trends within and between different solutions. Further, SolvationAnalysis is designed with extensibility in mind, exposing a core representation of solvation that users can use for brand new analyses. To get started with using `SolvationAnalysis` in your workflow, check out the documentation and tutorials on [readthedocs][docs].

^^ Couple extra lines describing use, feel free to edit!

**Review by orbeckst — APPROVED** [review]
I included @orionarcher 's update and added his summary image (maybe should be scaled... but we can fix that when we see it in the site) and links.

@jennaswa when you're happy with it, please squash-merge.

**Pull request merged**